docs/getting-started/backends-and-brokers/index.rst

```diff
@@ -76,10 +76,12 @@ RabbitMQ
 
 RabbitMQ is a broker.
 
-RabbitMQ handles larger messages better than Redis, however if many messages are coming in very quickly, scaling can become a concern and Redis or SQS should be considered unless RabbitMQ is running at very large scale.
+**As a Broker:** RabbitMQ handles larger messages better than Redis, however if many messages are coming in very quickly, scaling can become a concern and Redis or SQS should be considered unless RabbitMQ is running at very large scale.
 
 :ref:`See documentation for details <broker-rabbitmq>`
 
+**As a Backend:** RabbitMQ can store results via ``rpc://`` backend. This backend creates separate temporary queue for each new result of a task call. Creating new queue for each task call can be a bottleneck for high volume usage.
```

Comment:
I don't believe this description is accurate. The old AMQP backend created a queue per task call, while the RPC backend creates a queue per task caller (i.e. each client gets a separate queue that all results go into).

Reply:
Yes. I did mistake and already fixed it.

Reply:
I saw! The new description makes sense. Thank you!